src/components/MapSearchBar/MapSearchBar.css

```diff
@@ -0,0 +1,81 @@
+.map-search-panel {
+  position: absolute;
+  right: 0;
+  margin: 20px;
```

Comment:
Let's set the margin to match either the zoom-in/zoom-out buttons or the segment border in the right panel